aiida_workgraph/properties/builtins.py

```diff
@@ -1,11 +1,91 @@
 from typing import Dict, List, Union, Callable
-from node_graph.property import NodeProperty
+from aiida_workgraph.property import TaskProperty
 from node_graph.serializer import SerializeJson
 from node_graph.properties.builtins import PropertyVector, PropertyAny
 from aiida import orm
 
 
-class PropertyAiiDAInt(NodeProperty, SerializeJson):
+class PropertyInt(TaskProperty, SerializeJson):
+    """A new class for integer type."""
+
+    identifier: str = "workgraph.int"
+    data_type = "Int"
+
+    def __init__(self, name, description="", default=None, update=None) -> None:
+        super().__init__(name, description, default, update)
+
+    def set_value(self, value):
+        # run the callback function
+        if isinstance(value, (int, type(None))):
+            self._value = value
+            if self.update is not None:
+                self.update()
+        else:
+            raise Exception("{} is not a integer.".format(value))
+
+
+class PropertyFloat(TaskProperty, SerializeJson):
+    """A new class for float type."""
+
+    identifier: str = "workgraph.float"
+    data_type = "Float"
```

Comment:
Is this uppercase because it denotes an `orm.Float`?

Reply:
No special meaning, the `data_type` is case-insensitive, so it doesn't matter.